lib/kernel.nix

```diff
@@ -1,6 +1,14 @@
 { lib }:
 
-with lib;
+let
+  inherit (lib)
+    mkIf
+    optional
```

Comment:
"lib.optional" is not used in this file, "optional" is only used as a attribute name in an attrset

Reply:
Dang it, I usually catch these. [My script](https://gist.github.com/philiptaron/87b5889efb90d0a35809669d1d3d8816) produces false positives like this, but (thankfully) not false negatives, which would be much worse.